### PR TITLE
Usability updates to various Event GUI components

### PR DESCRIPTION
**Pull request by smbende1**

### CODE REVIEW

File: src/net/sf/memoranda/ui/EventDialog.java [hunk 1]
```diff
@@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
     }
 
     void okB_actionPerformed(ActionEvent e) {
-        this.dispose();
+    	if (textField.getText().isEmpty())
+    	{
+    		JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");
```
Comment (krcarey1) [inline src/net/sf/memoranda/ui/EventDialog.java:556]: I thought we were doing { at the end of a line and } on its own line per the CG? If yes, then all reference in this PR should be changed.

Comment (smbende1) [inline src/net/sf/memoranda/ui/EventDialog.java:556]: I'll check - I thought for loops it was specified that braces would be on a line.

On Wed, Feb 10, 2016 at 12:01 AM -0800, "krcarey1" <notifications@github.com<mailto:notifications@github.com>> wrote:

In src/net/sf/memoranda/ui/EventDialog.javahttps://github.com/cst316/spring16project-Team-El-Paso/pull/19#discussion_r52424596:

> @@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
>      }
> 
> ```
>  void okB_actionPerformed(ActionEvent e) {
> ```
> -        this.dispose();
> -     if (textField.getText().isEmpty())
> -     {
> -             JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");

I thought we were doing { at the end of a line and } on its own line per the CG?

##

Comment (smbende1) [inline src/net/sf/memoranda/ui/EventDialog.java:556]: You are right, of course. I'll make an amended commit with some cleanups.

On Wed, Feb 10, 2016 at 12:01 AM -0800, "krcarey1" <notifications@github.com<mailto:notifications@github.com>> wrote:

In src/net/sf/memoranda/ui/EventDialog.javahttps://github.com/cst316/spring16project-Team-El-Paso/pull/19#discussion_r52424596:

> @@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
>      }
> 
> ```
>  void okB_actionPerformed(ActionEvent e) {
> ```
> -        this.dispose();
> -     if (textField.getText().isEmpty())
> -     {
> -             JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");

I thought we were doing { at the end of a line and } on its own line per the CG?

##

**Comment by smbende1** [comment]
Updated with some stylistic changes.

**Comment by krcarey1** [comment]
I have no idea what these failures are. "The Travis CI build failed"

**Comment by smbende1** [comment]
Think it has to do with whatever we are expected to do for next Sprint - some code inspection tool I think.

**Comment by krcarey1** [comment]
Not sure if the Travis failures are an issue here, but PR looks good. Merging.

**Comment by krcarey1** [comment]
Can't merge from phone with the errors, will try later.

**Pull request merged**